README.md

```diff
@@ -27,6 +27,23 @@ The MT-GenEval benchmark was released in the EMNLP 2022 paper [MT-GenEval: A Cou
 The data is originally sourced from Wikipedia. 
 We include sentence-level development and test segments in `data/sentences/` and inter-sentence test segments in `data/context/`. 
 
+## Compute accuracy
+To compute accuracy, call `accuracy_metric` function in `accuracy_metric.py`. Example usage for English-Russian contextual dataset is as follows
+```
+hypothesis_path = PATH_FOR_YOUR_SYSTEM_TRANSLATIONS
+references_path = 'data/context/geneval-context-wikiprofessions-original-test.en_ru.ru'
+flipped_references_path = 'data/context/geneval-context-wikiprofessions-flipped-test.en_ru.ru'
+accuracy, metric_decisions = accuracy_metric(hypothesis_path, references_path, flipped_references_path)
+```
+For counterfactual dataset (the data in `data/sentences/`), metric decision can be obtained similarly but the accuracy computation is a little different as mentioned in Sec 3.1 of the paper. We consider a segment `Correct` only if both the original and the counterfactual segments are `Correct`. More specifically, 
+```
+_, metric_decisions_masculine = accuracy_metric(hypothesis_masculine_path, masculine_references_path, female_references_path)
```

Comment:
nit: let's be consistent on `feminine`/`masculine` vs. `female`/`male`. It looks like the rest of the code uses the former.